wire-ios-system/Source/ZMSLog.swift

```diff
@@ -278,6 +280,14 @@ extension ZMSLog {
         }
     }
 
+    //clears cache directory
+    @objc public static func clearCacheDirectory() {
+        guard let cachesDirectoryPath = cachesDirectory else { return }
+            closeHandle()
+            let manager = FileManager.default
+            try? manager.removeItem(at: cachesDirectoryPath)
+    }
```

Comment:
I don't see why this should be part of `ZMSLog`, I suggest to move it somewhere more appropriate, even the `SessionManager` or the `ZMUserSession`.

Reply:
Moved clear cache implementation to SessionManager